### PR TITLE
docs: re-organized README

### PR DESCRIPTION
**Pull request by alexgarel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,71 @@
...
+
+## 🔬 Projects
+
+Here are different experiments.
+
+Nutrition table:
+
+* [Nutrition table detection and extraction (2018 GSoc work by Sagar)](./GSoC2018/table_detection) - integrated in Robotoff, used for the detection part by the Graphnet and TableNet models
+* [Nutrition Table Extraction (2020 by Sadok, Yichen and Ramzi)](./nutrition-table-extraction/data_exploration/README.md) - on Graphnet and TableNet
+
```
Comment (teolemon) [inline README.md]: Basic nutrition extraction on the Robotoff side

File: README.md [hunk 2]
```diff
@@ -1,28 +1,71 @@
...
+* not deployed:
+
+  * [EM Lyon Category prediction (2020)](./ai-emlyon/README.md)  - not yet evaluated and integrated
+  * [Category from OCR prediction, Laure (Laurel16) (2021)](https://github.com/Laurel16/OpenFoodFactsCategorizer) - not yet evaluated and integrated - Categories maybe too general
+
+Logos:
+
+* [Labels and Logo detection (Data 4 Good, by Raphael, Charlotte and Antoine](./data4good_logo_detection/README.md) - code is duplicated and integrated in Robotoff
+* logo-ann (related to logos and labels) - classification using approximate KNN search - deployed in [robotoff-ann](https://github.com/openfoodfacts/robotoff-ann)
+
```
Comment (teolemon) [inline README.md]: Add issue from Robotoff recently added by RAphael

**Review by teolemon — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,28 +1,71 @@
...
-* [Google.org fellowship (2021) - Category prediction based on ingredients and title](https://github.com/kulizhsy/off-category-classification/)
-* [Labels and Logo detection (Data 4 Good, by Raphael, Charlotte and Antoine, code is duplicated and integrated in Robotoff](https://github.com/openfoodfacts/openfoodfacts-ai/blob/develop/data4good_logo_detection/README.md)
-* [Nutrition Table Extraction (2020 by Sadok, Yichen and Ramzi, on Graphnet and TableNet)](https://github.com/openfoodfacts/openfoodfacts-ai/blob/develop/nutrition-table-extraction/data_exploration/README.md)
-* [Spellcheck (by Wauplin, code is duplicated and integrated in Robotoff)](https://github.com/openfoodfacts/openfoodfacts-ai/blob/develop/spellcheck/README.md)
-* logo-ann (related to logos and labels)
+* Most trained Models and useful datasets are attached to [releases of this project](https://github.com/openfoodfacts/openfoodfacts-ai/releases) or [releases on robotoff-models](https://github.com/openfoodfacts/robotoff-models/releases).
+
+  [A google spreasheet](https://docs.google.com/spreadsheets/d/1p2tvA5ySm0RJpTjUwT3fFrDJNJLXNlVTkxxU-izTIMA/edit#gid=0) also tracks active models.
```
Comment (teolemon) [inline README.md]:
```suggestion
  [A Google spreadsheet](https://docs.google.com/spreadsheets/d/1p2tvA5ySm0RJpTjUwT3fFrDJNJLXNlVTkxxU-izTIMA/edit#gid=0) also tracks active models.
```

File: README.md [hunk 4]
```diff
@@ -1,28 +1,71 @@
...
+* not deployed:
+
+  * [EM Lyon Category prediction (2020)](./ai-emlyon/README.md)  - not yet evaluated and integrated
+  * [Category from OCR prediction, Laure (Laurel16) (2021)](https://github.com/Laurel16/OpenFoodFactsCategorizer) - not yet evaluated and integrated - Categories maybe too general
+
+Logos:
+
+* [Labels and Logo detection (Data 4 Good, by Raphael, Charlotte and Antoine](./data4good_logo_detection/README.md) - code is duplicated and integrated in Robotoff
+* logo-ann (related to logos and labels) - classification using approximate KNN search - deployed in [robotoff-ann](https://github.com/openfoodfacts/robotoff-ann)
+
```
Comment (teolemon) [inline README.md]:
```suggestion
* Updating the pre-weighted model to recent publications offers a nice no-effort boost
```

File: README.md [hunk 5]
```diff
@@ -1,28 +1,71 @@
...
+## 🔬 Projects
+
+Here are different experiments.
+
+Nutrition table:
+
+* [Nutrition table detection and extraction (2018 GSoc work by Sagar)](./GSoC2018/table_detection) - integrated in Robotoff, used for the detection part by the Graphnet and TableNet models
+* [Nutrition Table Extraction (2020 by Sadok, Yichen and Ramzi)](./nutrition-table-extraction/data_exploration/README.md) - on Graphnet and TableNet
+
```
Comment (teolemon) [inline README.md]:
```suggestion
* [Nutrition Table Extraction (2020 by Sadok, Yichen and Ramzi)](./nutrition-table-extraction/data_exploration/README.md) - on Graphnet and TableNet
* Basic nutrition extraction for text tables, already in the Robotoff API
```

**Comment by alexgarel** [comment]
Cool thanks @teolemon ! That's team work :grin:

**Pull request merged**